HHS/OPRE-OPS
# Refine CANs pt 2

## What changes

- Displays division as "DCFD", not "DCFC"
- "Source" is renamed "Funding Source"
- "Agencies" is renamed to "Funding Partners"

## Issue

#199 

## How to test

- Pull down branch
- Build locally
- Navigate to `CANs` page and see changes named above

## Screenshots
<img width="1430" alt="Screen Shot 2021-10-05 at 12 46 56 PM" src="https://user-images.githubusercontent.com/10871517/136432803-d2d26128-eb65-47a6-9fd6-657e7741321a.png">
<img width="1438" alt="Screen Shot 2021-10-05 at 12 47 05 PM" src="https://user-images.githubusercontent.com/10871517/136432807-c668ae6f-2b70-435d-8afd-6b8ed0da824e.png">
